worlds/yoshisisland/setup_game.py

```diff
@@ -0,0 +1,462 @@
+        elif world.global_level_list[i] in panel_palette_2:
+            world.palette_panel_list.extend([0x00, 0x10])
+
+    world.palette_panel_list[16:16] = [0x00, 0x0c, 0x00, 0x0c, 0x00, 0x18, 0x00, 0x18]
+    world.palette_panel_list[40:40] = [0x00, 0x0c, 0x00, 0x0c, 0x00, 0x18, 0x00, 0x18]
+    world.palette_panel_list[64:64] = [0x00, 0x0c, 0x00, 0x0c, 0x00, 0x18, 0x00, 0x18]
+    world.palette_panel_list[88:88] = [0x00, 0x0c, 0x00, 0x0c, 0x00, 0x18, 0x00, 0x18]
+    world.palette_panel_list[112:112] = [0x00, 0x0c, 0x00, 0x0c, 0x00, 0x18, 0x00, 0x18]
+
+    world.level_gfx_table.insert(8, 0x15)
+    world.level_gfx_table.insert(8, 0x15)
+    world.level_gfx_table.insert(8, 0x15)
+    world.level_gfx_table.insert(8, 0x11)
+
+    world.level_gfx_table.insert(20, 0x15)
+    world.level_gfx_table.insert(20, 0x15)
+    world.level_gfx_table.insert(20, 0x15)
+    world.level_gfx_table.insert(20, 0x11)
+
+    world.level_gfx_table.insert(32, 0x15)
+    world.level_gfx_table.insert(32, 0x15)
+    world.level_gfx_table.insert(32, 0x15)
+    world.level_gfx_table.insert(32, 0x11)
+
+    world.level_gfx_table.insert(44, 0x15)
+    world.level_gfx_table.insert(44, 0x15)
+    world.level_gfx_table.insert(44, 0x15)
+    world.level_gfx_table.insert(44, 0x11)
+
+    world.level_gfx_table.insert(56, 0x15)
+    world.level_gfx_table.insert(56, 0x15)
+    world.level_gfx_table.insert(56, 0x15)
+    world.level_gfx_table.insert(56, 0x15)
+
+
+    castle_door_dict = {
+                    0: [0xB8, 0x05, 0x77, 0x00],
+                    1: [0xB8, 0x05, 0x77, 0x00],
+                    2: [0xC6, 0x07, 0x7A, 0x00],
+                    3: [0xCD, 0x05, 0x5B, 0x00],
+                    4: [0xD3, 0x00, 0x77, 0x06],
+                    5: [0xB8, 0x05, 0x77, 0x00]
+    }
+
+    world.castle_door = castle_door_dict[world.options.bowser_door_mode.value]
+
+    world.world_1_stages = world.global_level_list[0:8]
+    world.world_2_stages = world.global_level_list[8:16]
+    world.world_3_stages = world.global_level_list[16:24]
+    world.world_4_stages = world.global_level_list[24:32]
+    world.world_5_stages = world.global_level_list[32:40]
+    world.world_6_stages = world.global_level_list[40:47]
+
+    world.world_1_stages.extend([0x08, 0x09])
+    world.world_2_stages.extend([0x14, 0x15])
+    world.world_3_stages.extend([0x20, 0x21])
+    world.world_4_stages.extend([0x2C, 0x2D])
+    world.world_5_stages.extend([0x38, 0x39])
+    world.world_6_stages.extend([0x43, 0x44, 0x45])
+
+    bowser_text_table = {
+        0: [0xDE, 0xEE, 0xDC, 0xDC, 0xE5], #Gween
+        1: [0xE7, 0xE0, 0xE5, 0xE2, 0xD0], #Pink
+        3: [0xEB, 0xDF, 0xF0, 0xD8, 0xE5], #Thyan
+        2: [0xF0, 0xDC, 0xEE, 0xEE, 0xE6], #Yewow
+        4: [0xE7, 0xEC, 0xDF, 0xE7, 0xE3], #puhpl
+        5: [0xD9, 0xEE, 0xE6, 0xEE, 0xE5], #Bwown
+        6: [0xEE, 0xDC, 0xDB, 0xD0, 0xD0], #Wed
+        7: [0xD9, 0xEE, 0xEC, 0xDC, 0xD0] #Bwue
+    }

```

Comment:
These aren't in order, also this looks like it could be a list instead of a dict, since doing bowser_text_table[1] would get you the line marked 1 in both cases.

Reply:
Oh, I should've noticed that one in my first review as well, lol.